packages/anu-vue/src/components/list-item/AListItem.tsx

```diff
@@ -81,10 +81,10 @@ export const AListItem = defineComponent({
 
     // useLayer
     const { styles, classes } = getLayerClasses(
-      computed(() => props.isActive ? props.color || 'primary' : undefined),
-      computed(() => props.isActive ? props.variant || 'light' : 'text'),
+      computed(() => props.color ?? props.isActive ? props.color || 'primary' : undefined),
+      computed(() => props.isActive ? (!props.variant && !props.color ? 'text' : props.variant || 'light') : 'text'),
       toRef(props, 'states'),
-      { statesClass: 'states:10' },
+      computed(() => ({ statesClass: props.isActive ? 'states:10' : 'hover:states:10' })),
```

Comment:
@ManUtopiK What does this line do?

Reply:
I mean what's the need for `hover:states:10'`?

Reply:
I don't remember exactly why it was for. But I struggled a little bit with states before #86. 

Also, there are still some improvements to do.

1. We lose the state of the `isActive` item when using together a `color` AND the variant `text`.

Basic example without color and text:

![image](https://user-images.githubusercontent.com/188172/207436485-ed0a36bf-dc19-4082-bd6c-b2ed01e55403.png)

With props `color="success" variant="text"`:

![image](https://user-images.githubusercontent.com/188172/207487897-c1be9d04-e17e-455a-9d93-88d5b967ddfc.png)
`Donut jujubes` isn't highlighted.

2. We can add color when there is no click handler.
```vue
<AListItem
  title="Send"
  color="success"
/>
```
`color` props do nothing here.